spec.md

```diff
@@ -295,6 +295,18 @@ However, in some circumstances, the CO may lose state (for example when the CO c
 The plugin should handle this as gracefully as possible.
 The error code `OPERATION_PENDING_FOR_VOLUME` may be returned by the plugin in this case (see general error code section for details).
 
+#### Field Size Limits
+
+CSI defines general size limits for fields of various types (see table below).
+The general size limit for a particular field may be overridden by specifying a different size limit in said field's description.
+The general size limits apply for messages generated by both COs and plugins.
+
+| Size       | Field Type          |
+|------------|---------------------|
+| 128 bytes  | string              |
+| 4 KiB      | map<string, string> |
+| 4 KiB      | repeated string     |
```

Comment:
As part of the change you did in #133, `node_ids` will be `repeated string` as well. If we limit this, we'll introduce a limit for the number of nodes this spec can handle. I'd suggest we remove this limitation here. Instead, explicitly mention that `mount_flags` has a limit of 4K bytes.